e2e-provision.js

```diff
@@ -87,28 +82,12 @@ let data = {
       aiInstrumentationKey: 'E2E_DIAGNOSTICS_AI_INSTRUMENTATION_KEY',
     },
     githubRepo: 'https://github.com/Azure-Samples/e2e-diagnostic-eventhub-ai-function',
-    githubBranch: 'auto',
+    githubBranch: 'rentu-update',
```

Comment:
Will `rentu-update` branch be merged to `master` branch later? If yes, this place does not need to change